apps/src/templates/teacherNavigation/LessonMaterialsContainer.tsx

```diff
@@ -17,20 +17,22 @@ type Lesson = {
   id: number;
   position: number;
   lessonPlanHtmlUrl: string;
+  standardsUrl: string;
+  vocabularyUrl: string;
   resources: {
     Teacher: {
       key: string;
       name: string;
       url: string;
-      downloadUrl: string | null;
+      downloadUrl?: string;
```

Comment:
You will see throughout here that I moved some of these props to optional props.  This felt like the cleaner code than passing in `null`.  the donwlodUrl in particular is used more for videos than other resources.